app/controllers/employments/EndEmploymentController.scala

```diff
@@ -215,6 +214,7 @@ class EndEmploymentController @Inject()(
 
   def endEmploymentPage: Action[AnyContent] = (authenticate andThen validatePerson).async { implicit request =>
     implicit val user: AuthedUser = request.taiUser
+    implicit val lang: Lang = request.lang
```

Comment:
I've had a look through the usages for lang and I think most of them come from the forms partials e.g. `app/views/includes/forms/textArea.scala.html`

If you remove the lang from one of these forms partial (whichever is easiest to test) and fix up the compilation errors, run it through a quick test and check if Welsh still works, we should be able to delete nearly all these implicit langs.